manifests/charts/istiod-remote/values.yaml

```diff
@@ -153,6 +153,9 @@ telemetry:
       logWindowDuration: "43200s"
 # Revision is set as 'version' label and part of the resource names when installing multiple control planes.
 revision: ""
+# Default revision specifies whether a given installation is the default installation to be
+# targeted for validation and injection
+defaultRevision: false
```

Comment:
@Monkeyanator could you explain why this is needed if I specify revision=default for config in line 155?

Reply:
Also, what if there is multiple defaults?

I think I got it, users can specify revision: 1.9, and set it to default.  Thoughts on configuring it as:
revision: "1.9"
  default: false|true
?

Reply:
Having multiple `defaultRevision`s will be an issue when using the in-cluster operator since they'll both try and reconcile the `istiod` service to point to themselves. We'll need to make one default and disable the other as atomically as possible to avoid this- this is more a limitation of the API than the impl unfortunately. We actually have this same issue for revision tags and will probably need a declarative API or CRD to do better there.

With other install methods, it'll just work since the base resources get overwritten once on install.

```yaml
revision: "1-9"
  default: false|true
```
looks good to me but I think we should keep it consistent with how it is in the `IstioOperator` API for the time being (which is currently [default_revision](https://github.com/istio/api/blob/35b7ba8ee9c54aed440d5e5b731b5d332b99b5d5/operator/v1alpha1/operator.proto#L98)at same level as `revision`).